sdk/src/slot_history.rs

```diff
@@ -65,14 +65,18 @@ impl SlotHistory {
     pub fn check(&self, slot: Slot) -> Check {
         if slot >= self.next_slot {
             Check::Future
-        } else if self.next_slot - slot > MAX_ENTRIES {
```

Comment:
Well, this causes panic if given slot is large enough. Separate pr....

Reply:
Well, I was lying... https://github.com/solana-labs/solana/pull/10718/files#diff-b6bc75c0bdcd2062ec00c3da3d974053R66 ensures that the overflow doesn't happen....